en/best-layout-practices.md

```diff
@@ -71,96 +43,57 @@ This section provides design guidance for the horizontal layout of Components an
 
 In order for a Component or Pattern to scale to the full width of the group or artboard it belongs to, make sure that its left and right side match with the group or artboard left and right side.
 
-**The Input will scale with the change of width in its parent. Since the left and right sides match with those of the group or artboard it belongs to, pinning both left and right will result in the same responsive behavior as when none of the sides are pinned.**
+**The Input will scale with the change of width in its parent. Since the left and right sides match with those of the group or artboard it belongs to, pinning both left and right will result in the same responsive behavior as when none of the sides is pinned.**
 
 <img class="responsive-img" src="images/one_horizontal_nofix_do1.png" srcset="images/one_horizontal_nofix_do1@2x.png 2x" />
 
 **The Input will keep a left and right margin and scale correctly when its parent width changes. However, it will be better to put the Input in a group, so that it has proper flexible width behavior and set the left and right margins on the group instead.**
 
 <img class="responsive-img" src="images/one_horizontal_nofix_caution.png" srcset="images/one_horizontal_nofix_caution@2x.png 2x" />
 
-**The Input will keep a left margin and proportionally scale when its parent width changes, resulting in bizzare responsive behaviors.**
+**The Input will keep a left margin and proportionally scale when its parent width changes, allowing responsive behavior.**
 
 <img class="responsive-img" src="images/one_horizontal_nofix_dont1.png" srcset="images/one_horizontal_nofix_dont1@2x.png 2x" />
 
-**The Input will proportionally scale and move left and right when its parent width changes, resulting in bizzare responsive behaviors.**
+**The Input will proportionally scale and move left and right when its parent width changes, allowing responsive behavior.**
 
 <img class="responsive-img" src="images/one_horizontal_nofix_dont2.png" srcset="images/one_horizontal_nofix_dont2@2x.png 2x" />
 
 ### Fixed Width
 
-In order for a Component or Pattern to keep its width, make sure that it has a fixed width and is pinned either to the left or right side of the group or artboard it belongs to. In most scenarios though, our advice is to use the flexible widths explained above for elements that appear alone on a layout row, unless these elements come with a fixed width by default as a component e.g. Avatar, Circular Progress Bar, Icon and Toast.
+In order for a Component or Pattern to keep its width, make sure that it has a fixed width in Sketch. In most scenarios though, our advice is to use the flexible widths explained above for elements that appear alone on a layout row, unless these elements come with a fixed width by default as a component e.g. Avatar, Circular Progress Bar, Icon and Toast.
```

Comment:
Only the Avatar is marked as resizable: false and will have fixed size always. All others - Icon, Toast, Circular Bar, would be resizable (width in %) unless explicitly fixed.